src/utils.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
```

Comment:
I tend to dislike the pattern where you have a `misc` package where you push all the things that really don't belong anywhere. Modules are cheap in JS, so why not create one named `encoding`, just for this? :)

Reply:
so i can erase that ?

Reply:
it's not related to this `use strict` line per se. It was more a comment on the name of the file..

Reply:
ok i can rename  'encoding.js'

Reply:
👍